.github/workflows/compile-check.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+on: [push]
+
+jobs:
+  build:
+    name: Contract compile check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Node.js 18.18
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.18
+
+      - name: Run Era Test Node
+        uses: dutterbutter/era-test-node-action@latest
+
+      - name: Setup project dependencies 
+        run: |
+          yarn global add node-gyp@10.0.1 && yarn install
+
+      - name: Compile and deploy ExecutionHelper.
+        working-directory: ./ExecutionHelper
+        run: |
+          yarn install
+          yarn hardhat compile
+          yarn hardhat deploy-zksync --script deploy.ts --network inMemoryNode
```

Comment:
We should split these steps and use our makefile. Maybe we can do that in another PR

Reply:
Yeah, I think we can do this right here, let me check

Reply:
MMm, that did not turned out as I was expecting it, lets check it on another PR, unless you have a clear idea on how to approach this?